File: src/main/java/Api/model/statisticReturn.java
package Api.model;

public class statisticReturn {

    private String employee;
    private String project;
    private String subproject;
    private double hours;

    public String getEmployee() {
        return employee;
    }

    public void setEmployee(String employee) {
        this.employee = employee;
    }

    public String getProject() {
        return project;
    }

    public void setProject(String project) {
        this.project = project;
    }

    public String getSubproject() {
        return subproject;
    }

    public void setSubproject(String subproject) {
        this.subproject = subproject;
    }

    public double getHours() {
        return hours;
    }

    public void setHours(double hours) {
        this.hours = hours;
    }


}

File: Client/src/app/models/StatisticReturn.ts
export class StatisticReturn
{
  constructor(
    private _employee?: string,
    private _project?: string,
    private _subproject?: string,
    private _hours?: number,
    private _minutes?: number) {
  }

  public get employee(): string {
    return this._employee;
  }

  public set employee(value: string) {
    this._employee = value;
  }

  public get project(): string {
    return this._project;
  }

  public set project(value: string) {
    this._project = value;
  }

  public get subproject(): string {
    return this._subproject;
  }

  public set subproject(value: string) {
    this._subproject = value;
  }

  public get hours(): number {
    return this._hours;
  }

  public set hours(value: number) {
    this._hours = value;
  }

  public get minutes(): number {
    return this._minutes;
  }

  public set minutes(value: number) {
    this._minutes = value;
  }
}

